README.md

```diff
@@ -10,27 +9,17 @@
 
 Node/Browser Javascript library for ArcGIS
 
-## Install
-
 ```
 npm install arcgis
 ```
 
-## Usage
-
```

Comment:
Drive-by comments from a quitter:

1. I think having `Install` and `Usage` sections up front is a good pattern worth keeping (and expanding).
2. https://github.com/Esri/node-arcgis/blob/lic/LICENSE.md and https://github.com/Esri/node-arcgis/blob/lic/license.txt conflict, you probably want to delete LICENSE.md.